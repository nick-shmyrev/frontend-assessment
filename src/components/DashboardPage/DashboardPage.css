.dashboard {
  background: #fff;
  border-radius: 10px;
  border: 1px solid #ccc;
  width: 80vw;
  height: 80vh;
  position: relative;
  padding-top: 50px;
  overflow: hidden;
}

.list-container {
  height: 100%;
  overflow-y: scroll;
}
