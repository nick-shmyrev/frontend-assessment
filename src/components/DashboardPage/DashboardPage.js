import React from 'react';

// Import components
import StudentFilter from '../StudentFilter/StudentFilter';
import StudentsList from '../StudentsList/StudentsList';

// Import styles
import style from './DashboardPage.css';

const HatchwaysApp = () => (
  <div className={style.dashboard}>
    <StudentFilter style={style.filters}/>
    <div className={style['list-container']}>
      <StudentsList/>
    </div>
  </div>
);

export default HatchwaysApp;
