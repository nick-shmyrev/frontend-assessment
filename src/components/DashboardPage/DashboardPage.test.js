import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from './DashboardPage';

describe('DashboardPage component', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<DashboardPage/>);
    expect(wrapper).toMatchSnapshot();
  });
});
