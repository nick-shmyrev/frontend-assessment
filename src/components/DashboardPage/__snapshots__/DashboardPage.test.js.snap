// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardPage component should render correctly 1`] = `
<div
  className="dashboard"
>
  <Connect(StudentFilter)
    style="filters"
  />
  <div
    className="list-container"
  >
    <Connect(StudentsList) />
  </div>
</div>
`;
